<HR COLOR=red>
【ExceptionTime】：2016-12-19 20:11:12,505 [4068] <BR>
【ExceptionLevel】：ERROR <BR>
【ExceptionClassName】：logerror [(null)] <BR>
【AttachedInfo】 : <br/><strong>客户机IP</strong>：::1<br /><strong>错误地址</strong>：http://localhost:62856/<br>【ExceptionType】：SqlException <br>【ExceptionMsg】：已成功与服务器建立连接，但是在登录前的握手期间发生错误。 (provider: SSL Provider, error: 0 - 接收到的消息异常，或格式不正确。) <br>【CallStack】：   在 System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)<br>   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)<br>   在 System.Data.SqlClient.TdsParser.ConsumePreLoginHandshake(Boolean encrypt, Boolean trustServerCert, Boolean integratedSecurity, Boolean& marsCapable)<br>   在 System.Data.SqlClient.TdsParser.Connect(ServerInfo serverInfo, SqlInternalConnectionTds connHandler, Boolean ignoreSniOpenTimeout, Int64 timerExpire, Boolean encrypt, Boolean trustServerCert, Boolean integratedSecurity, Boolean withFailover)<br>   在 System.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)<br>   在 System.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)<br>   在 System.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)<br>   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData)<br>   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)<br>   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)<br>   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)<br>   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)<br>   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)<br>   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)<br>   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)<br>   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)<br>   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)<br>   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)<br>   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)<br>   在 System.Data.SqlClient.SqlConnection.Open()<br>   在 DAL.SqlServerDal.PrepareCommand(SqlCommand cmd, SqlConnection conn, SqlTransaction trans, CommandType cmdType, String cmdText, List`1 cmdParms) 位置 e:\1.临时工作区\1.PubProj_MvcWebApp\DAL\SqlServerDal.cs:行号 23<br>   在 DAL.SqlServerDal.QuerySql(String sql) 位置 e:\1.临时工作区\1.PubProj_MvcWebApp\DAL\SqlServerDal.cs:行号 52<br>   在 MvcWebApp.Controllers.HomeController.Index() 位置 e:\1.临时工作区\1.PubProj_MvcWebApp\MyMvcWebApp\Controllers\HomeController.cs:行号 32<br>   在 lambda_method(Closure , ControllerBase , Object[] )<br>   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)<br>   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)<br>   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass42.<BeginInvokeSynchronousActionMethod>b__41()<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass8`1.<BeginSynchronous>b__7(IAsyncResult _)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass37.<>c__DisplayClass39.<BeginInvokeActionMethodWithFilters>b__33()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass4f.<InvokeActionMethodFilterAsynchronously>b__49()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass37.<BeginInvokeActionMethodWithFilters>b__36(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<>c__DisplayClass2a.<BeginInvokeAction>b__20()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<BeginInvokeAction>b__22(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeAction(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Controller.<>c__DisplayClass1d.<BeginExecuteCore>b__18(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Controller.EndExecuteCore(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Controller.EndExecute(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Controller.System.Web.Mvc.Async.IAsyncController.EndExecute(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.MvcHandler.<>c__DisplayClass8.<BeginProcessRequest>b__3(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.MvcHandler.EndProcessRequest(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.MvcHandler.System.Web.IHttpAsyncHandler.EndProcessRequest(IAsyncResult result)<br>   在 System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()<br>   在 System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) <BR>
 <HR Size=1><HR COLOR=red>
【ExceptionTime】：2016-12-19 20:23:03,887 [1840] <BR>
【ExceptionLevel】：ERROR <BR>
【ExceptionClassName】：logerror [(null)] <BR>
【AttachedInfo】 : <br/><strong>客户机IP</strong>：::1<br /><strong>错误地址</strong>：http://localhost:62856/System/UserManagerSave?time=0.17508448148146272<br>【ExceptionType】：WebException <br>【ExceptionMsg】：未能解析此远程名称: 'myportal.wneweb.com.tw' <br>【CallStack】：   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)<br>   在 System.Net.HttpWebRequest.GetRequestStream()<br>   在 WsPlugin.DynamicWS.Invoke(Boolean encode)<br>   在 WsPlugin.DynamicWS.Invoke()<br>   在 MvcWebApp.Controllers.SystemController.CheckEmpNo(String empNo) 位置 e:\1.临时工作区\1.PubProj_MvcWebApp\MyMvcWebApp\Controllers\SystemController.cs:行号 153<br>   在 MvcWebApp.Controllers.SystemController.UserManagerSave(FormCollection fc) 位置 e:\1.临时工作区\1.PubProj_MvcWebApp\MyMvcWebApp\Controllers\SystemController.cs:行号 136<br>   在 lambda_method(Closure , ControllerBase , Object[] )<br>   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)<br>   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)<br>   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass42.<BeginInvokeSynchronousActionMethod>b__41()<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass8`1.<BeginSynchronous>b__7(IAsyncResult _)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass37.<>c__DisplayClass39.<BeginInvokeActionMethodWithFilters>b__33()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass4f.<InvokeActionMethodFilterAsynchronously>b__49()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass37.<BeginInvokeActionMethodWithFilters>b__36(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<>c__DisplayClass2a.<BeginInvokeAction>b__20()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<BeginInvokeAction>b__22(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeAction(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Controller.<>c__DisplayClass1d.<BeginExecuteCore>b__18(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Controller.EndExecuteCore(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Controller.EndExecute(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Controller.System.Web.Mvc.Async.IAsyncController.EndExecute(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.MvcHandler.<>c__DisplayClass8.<BeginProcessRequest>b__3(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.MvcHandler.EndProcessRequest(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.MvcHandler.System.Web.IHttpAsyncHandler.EndProcessRequest(IAsyncResult result)<br>   在 System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()<br>   在 System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) <BR>
 <HR Size=1><HR COLOR=red>
【ExceptionTime】：2016-12-19 20:23:23,947 [5172] <BR>
【ExceptionLevel】：ERROR <BR>
【ExceptionClassName】：logerror [(null)] <BR>
【AttachedInfo】 : <br/><strong>客户机IP</strong>：::1<br /><strong>错误地址</strong>：http://localhost:62856/System/AjaxEmpInfo<br>【ExceptionType】：WebException <br>【ExceptionMsg】：未能解析此远程名称: 'myportal.wneweb.com.tw' <br>【CallStack】：   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)<br>   在 System.Net.HttpWebRequest.GetRequestStream()<br>   在 WsPlugin.DynamicWS.Invoke(Boolean encode)<br>   在 WsPlugin.DynamicWS.Invoke()<br>   在 MvcWebApp.Controllers.SystemController.AjaxEmpInfo() 位置 e:\1.临时工作区\1.PubProj_MvcWebApp\MyMvcWebApp\Controllers\SystemController.cs:行号 194<br>   在 lambda_method(Closure , ControllerBase , Object[] )<br>   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)<br>   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)<br>   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass42.<BeginInvokeSynchronousActionMethod>b__41()<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass8`1.<BeginSynchronous>b__7(IAsyncResult _)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass37.<>c__DisplayClass39.<BeginInvokeActionMethodWithFilters>b__33()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass4f.<InvokeActionMethodFilterAsynchronously>b__49()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass37.<BeginInvokeActionMethodWithFilters>b__36(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<>c__DisplayClass2a.<BeginInvokeAction>b__20()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<BeginInvokeAction>b__22(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeAction(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Controller.<>c__DisplayClass1d.<BeginExecuteCore>b__18(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Controller.EndExecuteCore(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Controller.EndExecute(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Controller.System.Web.Mvc.Async.IAsyncController.EndExecute(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.MvcHandler.<>c__DisplayClass8.<BeginProcessRequest>b__3(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.MvcHandler.EndProcessRequest(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.MvcHandler.System.Web.IHttpAsyncHandler.EndProcessRequest(IAsyncResult result)<br>   在 System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()<br>   在 System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) <BR>
 <HR Size=1><HR COLOR=red>
【ExceptionTime】：2016-12-19 20:23:35,984 [10056] <BR>
【ExceptionLevel】：ERROR <BR>
【ExceptionClassName】：logerror [(null)] <BR>
【AttachedInfo】 : <br/><strong>客户机IP</strong>：::1<br /><strong>错误地址</strong>：http://localhost:62856/System/UserManagerSave?time=0.9412928021047264<br>【ExceptionType】：WebException <br>【ExceptionMsg】：未能解析此远程名称: 'myportal.wneweb.com.tw' <br>【CallStack】：   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)<br>   在 System.Net.HttpWebRequest.GetRequestStream()<br>   在 WsPlugin.DynamicWS.Invoke(Boolean encode)<br>   在 WsPlugin.DynamicWS.Invoke()<br>   在 MvcWebApp.Controllers.SystemController.CheckEmpNo(String empNo) 位置 e:\1.临时工作区\1.PubProj_MvcWebApp\MyMvcWebApp\Controllers\SystemController.cs:行号 153<br>   在 MvcWebApp.Controllers.SystemController.UserManagerSave(FormCollection fc) 位置 e:\1.临时工作区\1.PubProj_MvcWebApp\MyMvcWebApp\Controllers\SystemController.cs:行号 136<br>   在 lambda_method(Closure , ControllerBase , Object[] )<br>   在 System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)<br>   在 System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)<br>   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass42.<BeginInvokeSynchronousActionMethod>b__41()<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass8`1.<BeginSynchronous>b__7(IAsyncResult _)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass37.<>c__DisplayClass39.<BeginInvokeActionMethodWithFilters>b__33()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass4f.<InvokeActionMethodFilterAsynchronously>b__49()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass37.<BeginInvokeActionMethodWithFilters>b__36(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<>c__DisplayClass2a.<BeginInvokeAction>b__20()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<BeginInvokeAction>b__22(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeAction(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Controller.<>c__DisplayClass1d.<BeginExecuteCore>b__18(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Controller.EndExecuteCore(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.Controller.EndExecute(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Controller.System.Web.Mvc.Async.IAsyncController.EndExecute(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.MvcHandler.<>c__DisplayClass8.<BeginProcessRequest>b__3(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.<>c__DisplayClass4.<MakeVoidDelegate>b__3(IAsyncResult ar)<br>   在 System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`1.End()<br>   在 System.Web.Mvc.MvcHandler.EndProcessRequest(IAsyncResult asyncResult)<br>   在 System.Web.Mvc.MvcHandler.System.Web.IHttpAsyncHandler.EndProcessRequest(IAsyncResult result)<br>   在 System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()<br>   在 System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) <BR>
 <HR Size=1>